### ISSUE TITLE
Adhere to the XDG standard

### ISSUE THREAD

**Issue by Mailaender** [OpenRCT2/OpenRCT2#14402]
**OS:** @openSUSE 15.2
**Version:** 0.3.1
**Commit/Build:** lp152.14.3

My screenshot gets dumped into `~/.config/OpenRCT2/screenshot/` but I would expect something like `~/.local/share/openrct2/screenshot/` (compare @OpenTTD behavior)

- [ ] Reproducible in RCT2 (vanilla)?
- [ ] Specific to multiplayer?

**Steps to reproduce:**
Take a screenshot in game.

**Comment by Gymnasiast** [comment]
Personally, I would expect them to end up in `~/Pictures`. (Or more precisely, whatever the XDG environment variable for the pictures directory is.)
And one could say something similar about created parks - they're user-created files, surely those do not belong in `~/.local/share` either.

I suppose the track and scenario indices should go in `~/.cache/OpenRCT2` if we were to actually adhere to the XDG standard. And, on Windows, in `%localappdata%/OpenRCT2`.

**Comment by Mailaender** [comment]
You are right `$XDG_PICTURES_DIR` might make more sense from a user point of view. Never saw that in the wild yet. `$XDG_DATA_HOME` is for user data like tracks/parks. `$XDG_CACHE_HOME` should probably only be used for log files etc.

**Comment by Gymnasiast** [comment]
> `$XDG_CACHE_HOME ` is for non-essential files. I wouldn't put user creations there.

I wasn't proposing that. Note that I said "track and scenario **indices**".

**Comment by Gymnasiast** [comment]
This issue was created for the screenshots: https://github.com/OpenRCT2/OpenRCT2/issues/6233

Keeping this one open for the rest. I can agree on putting the cache in `$XDG_CACHE_HOME`, not quite sure about the rest.

**Comment by Thaodan** [comment]
> You are right `$XDG_PICTURES_DIR` might make more sense from a user point of view. Never saw that in the wild yet. `$XDG_DATA_HOME` is for user data like tracks/parks. `$XDG_CACHE_HOME` is for non-essential files. I wouldn't put user creations there.

It is read through the user-dir.dirs file:
https://man.archlinux.org/man/user-dirs.dirs.5.en

**Comment by Thaodan** [comment]
> Personally, I would expect them to end up in `~/Pictures`. (Or more precisely, whatever the XDG environment variable for the pictures directory is.) And one could say something similar about created parks - they're user-created files, surely those do not belong in `~/.local/share` either.
> 
> I suppose the track and scenario indices should go in `~/.cache/OpenRCT2` if we were to actually adhere to the XDG standard. And, on Windows, in `%localappdata%/OpenRCT2`.

Did https://github.com/OpenRCT2/OpenRCT2/pull/6748 handle that?
